app/src/main/java/com/bignerdranch/android/criminalintent/CrimeFragment.kt

```diff
@@ -106,7 +119,32 @@ class CrimeFragment: Fragment(), DatePickerFragment.Callbacks {
                 show(this@CrimeFragment.getParentFragmentManager(), DIALOG_DATE)
             }
         }
+
+        reportButton.setOnClickListener {
+            Intent(Intent.ACTION_SEND).apply {
+                type = "text/plain"
```

Comment:
You could put `"text/plain"` into a `private const val TYPE_PLAIN_TEXT` and store it in the companion object in this class. It's generally better **not to** hardcode any strings in the code, unless they are constant variables

Reply:
You're right.
Added `private const val TYPE_PLAIN_TEXT` to the existing companion object.